public/js/log.js

```diff
@@ -60,250 +59,295 @@ MAGI_log.resizes = {
 };
 MAGI_log.tooltips = [];
 
-$('#magi-loggingReadMore').click(function(e) {
-  e.preventDefault();
-  var isVisible = 'visible' == $('#magi-loggingReadMoreDiv').css('visibility'),
-      visibleState = isVisible ? 'hidden' : 'visible',
-      displayState = isVisible ? 'none' : 'block';
-
-  $('#magi-loggingReadMoreDiv').css('visibility', visibleState);
-  $('#magi-loggingReadMoreDiv').css('display', displayState);
-
+$().ready(function () {
+  $.get('/logEnabled', function(res, status) {
```

Comment:
This is on a case-by-case basis rather than a server configuration, so that people can opt out of being logged should they choose.